### DISCUSSION TITLE
amamba/user-guide/pipeline/create/custom/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1775] [Announcements]
# amamba/user-guide/pipeline/create/custom/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/amamba/user-guide/pipeline/create/custom/

**Comment by liyuerich** [comment]
创建流水线还有这个选项：创建多分支流水线。 能介绍一下吗？

**Reply by liyuerich** [reply]
文档站上看到了，自定义创建的图是旧的
